tensorflow_serving/model_servers/main.cc

```diff
@@ -101,6 +101,17 @@ using tensorflow::serving::PredictionService;
 
 namespace {
 
+void ParseProtoTextFile(const string& file, google::protobuf::Message* message) {
```

Comment:
nit: how about having this function return Status, and the callers can use QCHECK if they want.